scripts/sync-eslint-to-root.js

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env node
```

Comment:
This looks good, but I am concerned about us having to maintain this script here.

Reply:
@vmarta instead of this, would it be possible to have an `.eslintrc.js` at the root of the repo which imports the eslint from the `packages` directory?

Reply:
@alexvuong re: maintenance, I've stripped down the script logic and made it much simpler. No more fancy syncing, but just a simple check to remind us if some dependencies are missing from the root package.json.